src/06-positron-rag/lib/insert/prepareArticlesCollection.ts

```diff
@@ -110,7 +110,17 @@ export async function prepareArticlesCollection() {
         },
       },
       {
-        name: "head",
+        name: "sectionIndex",
+        dataType: ["int"],
```

Comment:
I persisted the section number here, but am not actually making use of it yet. 

(I've seen allusions to approaches that say something like _if you've retrieved over X% of the chunks in a document, then you might as well retrieve the document in its entirety_)